src/faq.md

```diff
@@ -144,27 +148,41 @@ appreciation through careful design choices and fair communication about them.
 
 ### Q. Is Dart a statically typed language?
 
-Dart is statically typed if you opt in to [strong mode]
-and use a compiler (such as the Dart Development Compiler ([DDC])),
+Dart 1.x is statically typed if you opt in to [strong mode]
+and use a compiler (such as the Dart Development Compiler
+([dartdevc]({{site.webdev}}/tools/dartdevc))),
 that has both static and runtime checks. In this environment, Dart
 has a sound type system, which guarantees that an expression of one
 type cannot produce a value of another type. (So, no surprises!)
-[Flutter]&mdash;a cross-platform mobile development system&mdash;uses
+Types won't be optional in Dart 2.0.
+[Flutter]&mdash;a cross-platform mobile development system&mdash;already uses
 strong mode.
 
 Even in strong mode, though, you can annotate any variable with
 `dynamic` if you need the flexibility of a dynamic language.
-`dynamic` can contain any type. Of course, that removes many of
-the benefits of a type-safe language for that variable.
+The `dynamic` type itself is static, but can contain any type at runtime.
+Of course, that removes many of the benefits of a type-safe language
+for that variable.
 
 ### Q. What is strong mode?
 
-Strong mode is Dart’s implementation of a sound type system.
+Strong mode contributes to Dart’s implementation of a sound type system.
 With strong mode enabled (in an implementation that has both the
 static and runtime checks), Dart is a statically typed language ensuring
 that static type annotations are actually correct at runtime.
 
-For more information, see [Strong Mode Dart].
+In Dart 1.x, you can [opt in to strong
+mode.](/guides/language/sound-dart#how-to-enable-strong-mode)
+We recommend that you migrate to strong mode now to take advantage of
+the new sound type system that finds errors at compile time.
```

Comment:
I don't like having "that finds errors at compile time" everywhere. It's bloaty!